src/wasm/wasm-emscripten.cpp

```diff
@@ -157,8 +162,7 @@ Function* EmscriptenGlueGenerator::generateMemoryGrowthFunction() {
 
 void EmscriptenGlueGenerator::generateStackInitialization(Address addr) {
   auto* stackPointer = getStackPointerGlobal();
-  if (stackPointer->imported())
-    Fatal() << "stack pointer not assignable becasue it is imported";
+  assert(!stackPointer->imported());
```

Comment:
Is the assert better than Fatal + error message?

Reply:
Talked in person: the Fatal is up above, the assert is re-affirming that as a precondition